src/Support/helpers.php

```diff
@@ -198,7 +198,22 @@ function plugins_path($path = '')
      */
     function uploads_path($path = '')
     {
-        return app('path.uploads').($path ? '/'.$path : $path);
+        $uploads_path = rtrim(Config::get('cms.storage.uploads.path', app('path.uploads')), '/');
+        return $uploads_path.($path ? '/'.ltrim($path, '/') : $path);
+    }
+}
+
+if (!function_exists('media_path')) {
+    /**
+     * Get the path to the media folder.
+     *
+     * @param  string  $path
+     * @return string
+     */
+    function media_path($path = '')
+    {
+        $media_path = rtrim(Config::get('cms.storage.media.path', app('path.media')), '/');
```

Comment:
@LukeTowers Should we be fetching `Config::get('cms.storage.media.path')` here or would it make more sense in Foundation\Application::mediaPath() ...

Reply:
Probably whatever the pattern is for the other helpers